src/seedu/addressbook/data/person/Address.java

```diff
@@ -34,25 +31,4 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
     public static boolean isValidAddress(String test) {
         return test.matches(ADDRESS_VALIDATION_REGEX);
     }
-
-    @Override
-    public String toString() {
-        return value;
-    }
-
-    @Override
-    public boolean equals(Object other) {
-        return other == this // short circuit if same object
-                || (other instanceof Address // instanceof handles nulls
-                && this.value.equals(((Address) other).value)); // state check
-    }
```

Comment:
`equals` should not be left to the parent class as objects from child classes need to be compared with their respective classes, and not with their parent class.